proto/runner.proto

```diff
@@ -36,14 +36,16 @@ message RunRequest {
   // Arch on which to run. Defaults to "amd64".
   string arch = 8;
 
-  // If true, start the runner but do not wait for the action to be scheduled.
-  bool async = 9;
```

Comment:
Are people using this? It might be safer to mark it as deprecated and convert it to a wait_until condition

Reply:
I did mark it as deprecated, I just moved it to the bottom of the message so it's more out of the way